README.md

```diff
@@ -1,27 +1,456 @@
 # Cordova Handpoint SDK Plugin
-# Still Work In Progress. Not ready for live use
+Cross-platform Handpoint SDK for Cordova. You might want to take a look at the [Ionic 2 starter application](https://github.com/handpoint/handpoint-ionic-app) to see a fully functional app using the plugin.
```

Comment:
Please, explain a bit further what this application is intended for.

For example:

"Cross-platform Handpoint application built over Cordova using the Handpoint SDK.

This app is intended to serve as a reference guide to build other applications using Handpoint SDK. You might want to take a look at the [Ionic 2 starter application](https://github.com/handpoint/handpoint-ionic-app) to see a fully functional app using the plugin.

No animal was harmed building this app."